install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -205,21 +205,20 @@ type SpiceDBConfig struct {
 }
 
 type WebAppConfig struct {
-	PublicAPI                  *PublicAPIConfig       `json:"publicApi,omitempty"`
-	Server                     *ServerConfig          `json:"server,omitempty"`
-	ProxyConfig                *ProxyConfig           `json:"proxy,omitempty"`
-	WorkspaceManagerBridge     *WsManagerBridgeConfig `json:"wsManagerBridge,omitempty"`
-	Tracing                    *Tracing               `json:"tracing,omitempty"`
-	UsePodAntiAffinity         bool                   `json:"usePodAntiAffinity"`
-	DisableMigration           bool                   `json:"disableMigration"`
-	Usage                      *UsageConfig           `json:"usage,omitempty"`
-	ConfigcatKey               string                 `json:"configcatKey"`
-	WorkspaceClasses           []WebAppWorkspaceClass `json:"workspaceClasses"`
-	Stripe                     *StripeConfig          `json:"stripe,omitempty"`
-	SlowDatabase               string                 `json:"slowDatabase,omitempty"`
-	IAM                        *IAMConfig             `json:"iam,omitempty"`
-	WithoutWorkspaceComponents bool                   `json:"withoutWorkspaceComponents,omitempty"`
```

Comment:
Diff is weird, but for future readers this bool field is removed.

Reply:
Hide whitespace gives you a proper diff view.

![image](https://user-images.githubusercontent.com/24960040/216297580-9f3cc8fb-a5d2-4528-9ec0-072a3be30e4a.png)

![image](https://user-images.githubusercontent.com/24960040/216297629-94bbef6f-a91e-49d4-839e-1cc2c468298f.png)